### DISCUSSION TITLE
Manually edit autograding.json

### DISCUSSION THREAD

**Discussion by alexandruradovici** [community/community#54147] [GitHub Education]
### Hi everyone,

I am using Github Classroom to assign and grade homework for a university class. The homework has around 100 test cases. The problem that I am facing is that I have to use the user interface and manually edit and add each test. I saw that this actually updates the `autograding.json` file in the student's repository.

Is there any way to edit directly this file?

**Comment by jzaldivar** [comment]
Yes. You can manually edit the `autograding.json` file. It's just a plain text file. Or, better yet, for hundreds of cases, generate it programmatically.

Just note that if you're plannning on editing the `autograding.json` file yourself, DO NOT enter any test case on assignment creation. Otherwise, GitHub Classroom will overwrite your `autograding.json` file when generating the assignment.

![image](https://github.com/community/community/assets/15006370/e7028400-1496-40cb-8cca-73280639adc9)
